### PR TITLE
Issue #104: Make conversion from lua objects to go a bit easier

### PR DESCRIPTION
**Pull request by gaffo**

### CODE REVIEW

**Comment by gaffo** [comment]
I'd like to discuss this with you before you merge...

File: value.go [hunk 1]
```diff
@@ -168,11 +180,11 @@ type LTable struct {
 	k2i     map[LValue]int
 }
 
-func (tb *LTable) String() string                     { return fmt.Sprintf("table: %p", tb) }
-func (tb *LTable) Type() LValueType                   { return LTTable }
-func (tb *LTable) assertFloat64() (float64, bool)     { return 0, false }
-func (tb *LTable) assertString() (string, bool)       { return "", false }
-func (tb *LTable) assertFunction() (*LFunction, bool) { return nil, false }
+func (tb LTable) String() string                     { return fmt.Sprintf("table: %p", tb) }
```
Comment (bbuck) [inline value.go]: What made you decide to copy `LTable` for all these method calls? I don't think removing the pointer receiver here is a good idea.

File: lua2go.go [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+		return value.(LString).String()
+	case LTFunction:
+		return nil
+	case LTTable:
+		return LuaTableToGo(value.(LTable))
+	}
+	return ""
+}
+
+func LuaTableToGo(t LTable) interface{} {
```
Comment (bbuck) [inline lua2go.go]: This doesn't handle circular references.

**Comment by gaffo** [comment]
They already said no on the issue request. Closing. I found some faster and better suited interpreted languages.

**Comment by bbuck** [comment]
Out of curiosity what language was that?

**Comment by gaffo** [comment]
I went with go-mruby, I'm much more familiar with it.

**Pull request closed without merging**